CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Upcoming]
+### Added
```

Comment:
<!--
  1 failure:  The latest date i...
  0 warning: 
  
  1 markdown notices
  DangerID: danger-id-Danger;
  File: CHANGELOG.md;
  Line: 8;
-->
- :no_entry_sign: The latest date in the CHANGELOG is not today!


```suggestion
## [0.22.0] - 2020-07-02
```